src/finder/frameworkSpecs.ts

```diff
@@ -26,82 +26,82 @@ export const frameworkSpecs: FrameworkSpec[] = [
     ],
     commands: {
       build: {
-        cmd: "ng build",
+        cmd: "npm run build",
       },
       dev: {
-        cmd: "ng run",
+        cmd: "npm run start",
         env: { NODE_ENV: "dev" },
       },
       run: {
-        cmd: "ng run",
+        cmd: "node server.js",
         env: { NODE_ENV: "production" },
       },
     },
   },
   {
-    id: "core:nextjs",
+    id: "nextjs",
     runtime: "nodejs",
-    requiredFiles: [["next.config.js", "next.config.ts"]],
+    requiredFiles: [["next.config.js"]],
```

Comment:
I think we still want to consider next.config.ts.

But then again, I'd be fine not including these frameworks in the initial pr. In fact, don't want to support some of the other framework until we are ready anyway.